proto/metapb.proto

```diff
@@ -61,6 +61,33 @@ message RegionEpoch {
     uint64 version = 2;
 }
 
+message BucketStats {
+    // total read (point get) in bytes of each bucket
+    repeated uint64 read = 1;
+
+    // total scan in bytes of each bucket
+    repeated uint64 query = 2;
+
+    // total write in bytes of each bucket
+    repeated uint64 write = 3;
+}
+
+message Buckets {
+    uint64 region_id = 1;
+
+    // A hint indicate if keys have changed.
+    uint64 version = 2;
+
+    // keys of buckets, not include start/end key of region
+    repeated bytes keys = 3;
+
+    // bucket stats
+    repeated BucketStats stats = 4;
```

Comment:
It's a bad idea. And it's explained in the RFC PR why the stats are flatten: https://github.com/tikv/rfcs/pull/82#discussion_r764879850

Reply:
Yeah, actually the repeated can be removed here. Then it's OK.